XmlFormatterOsIndependent/Commands/SwitchStyleCommand.cs

```diff
@@ -0,0 +1,63 @@
+﻿using Avalonia.Controls;
+using Avalonia.Markup.Xaml.Styling;
+using System;
+using System.Windows.Input;
+using XmlFormatterOsIndependent.DataSets;
+using XmlFormatterOsIndependent.Enums;
+
+namespace XmlFormatterOsIndependent.Commands
+{
+    /// <summary>
+    /// Switch the style of the current window
+    /// </summary>
+    class SwitchStyleCommand : ICommand
+    {
+        /// <summary>
+        /// The style to include for light theme
+        /// </summary>
+        private readonly StyleInclude light;
+
+        /// <summary>
+        /// The style to include for the dark theme
+        /// </summary>
+        private readonly StyleInclude dark;
+
+        /// <summary>
+        /// Create a new instance of this command class
+        /// </summary>
+        public SwitchStyleCommand()
+        {
+            light = new StyleInclude(new Uri("resm:Styles?assembly=ControlCatalog"))
+            {
+                Source = new Uri("resm:Avalonia.Themes.Default.Accents.BaseLight.xaml?assembly=Avalonia.Themes.Default")
+            };
+
+            dark = new StyleInclude(new Uri("resm:Styles?assembly=ControlCatalog"))
+            {
+                Source = new Uri("resm:Avalonia.Themes.Default.Accents.BaseDark.xaml?assembly=Avalonia.Themes.Default")
+            };
+        }
+
+        /// <inheritdoc/>
+        public event EventHandler CanExecuteChanged;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove the unused event 'CanExecuteChanged' or invoke it.](https://app.codacy.com/manual/simonaberle/XmlFormatter/pullRequest?prid=5634024)